### PR TITLE
add qt5::x11extras module

### PR DESCRIPTION
**Pull request by AndreyMlashkin**
Specify library name and version:  **qt/5.15.2**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/qt/5.x.x/conanfile.py [hunk 1]
```diff
@@ -1003,6 +1005,9 @@ def _create_plugin(pluginname, libname, type, requires):
         if self.options.qtnetworkauth:
             _create_module("NetworkAuth", ["Network"])
 
+        if self.options.qtx11extras:
```
Comment (SpaceIm) [inline recipes/qt/5.x.x/conanfile.py:1008]: use `get_safe`, since this option is deleted in `configure()` if build machine is not Linux.

**Review by ericLemanissier — APPROVED** [review]

File: recipes/qt/5.x.x/conanfile.py [hunk 2]
```diff
@@ -1003,6 +1005,9 @@ def _create_plugin(pluginname, libname, type, requires):
         if self.options.qtnetworkauth:
             _create_module("NetworkAuth", ["Network"])
 
+        if self.options.get_safe(qtx11extras):
```
Comment (SpaceIm) [inline recipes/qt/5.x.x/conanfile.py:1008]:
```suggestion
        if self.options.get_safe("qtx11extras"):
```

**Review by ericLemanissier — APPROVED** [review]

**Comment by AndreyMlashkin** [comment]
Has anybody a clue about this error? Is it a CI issue?

**Comment by ericLemanissier** [comment]
Yes, you can simply retrigger the build once again

**Review by prince-chrismc — APPROVED** [review]

File: recipes/qt/5.x.x/conanfile.py [hunk 3]
```diff
@@ -209,6 +209,8 @@ def configure(self):
         if tools.os_info.is_linux:
```
Comment (madebr) [inline recipes/qt/5.x.x/conanfile.py:209]: This should use `self.settings.os`.
```suggestion
        if self.settings.os in ("FreeBSD", "Linux"):
```

**Comment by uilianries** [comment]
This PR needs https://github.com/conan-io/conan-center-index/pull/6055 first

**Comment by ericLemanissier** [comment]
@AndreyMlashkin now that #6055 is merged, you can retrigger CI

**Comment by prince-chrismc** [comment]
Sync with master... this way with a the new commit sha it clean start a new build... tons of errors from one bad PR 😑

**Review by prince-chrismc — DISMISSED** [review]

**Review by ericLemanissier — APPROVED** [review]

**Review by prince-chrismc — APPROVED** [review]

**Comment by prince-chrismc** [comment]
🤞

**Comment by AndreyMlashkin** [comment]
Is there some system in failures or should I re-trigger the CI more times?

**Comment by prince-chrismc** [comment]
Excellent question... idk what's behind this error its usually transitive... it would be interesting to know why it failing repeatedly for this PR

>  ```
>   ERROR: PREV for bzip2/1.0.8:dac385276edd3ed845f4c6e5865148478cd26814 is None: {"recipe": {"revision": "9c8b632000a52af5af3f045d3a6db12f", "remote": "conan-center", "properties": {}, "checksums": {"conan_export.tgz": {"md5": "44e99ec27f446c8c2903d7ead4a7e8e4", "sha1": "5a2eb16ade9bced471bfe964bcc6a726c864fad6"}, "conanmanifest.txt": {"md5": "adc4d157f385f49917a7eced61df81f2", "sha1": "ef418bf7f259f20919e4bbf14da5d519d9d0b455"}, "conanfile.py": {"md5": "d56a339c2b00606410445db1ed559eb8", "sha1": "5e1f3b9d664cad1325b099f7e38f5acad18150a5"}}}, "packages": {"dac385276edd3ed845f4c6e5865148478cd26814": {"revision": null, "recipe_revision": null, "remote": null, "properties": {}, "checksums": {}}}}
>   ```

**Comment by AndreyMlashkin** [comment]
CI is still building the PR?...

**Comment by prince-chrismc** [comment]
It most likely failed to update the GitHub API with the failed status... Your PRs are haunted! 👻😈 

//cc @jgsogo @danimtb is there's something blocking the CI?

**Comment by prince-chrismc** [comment]
Yeah you need to wait for the infra team to take a look. I think we are the the point where cancelling the building is interrupting the next one and it's just stuck.

**Comment by madebr** [comment]
Can you try rebasing on top of current master + force pushing?

**Comment by prince-chrismc** [comment]
> Can you try rebasing on top of current master + force pushing?

I suggested that too... current commit was rebased

**Comment by AndreyMlashkin** [comment]
> Can you try rebasing on top of current master + force pushing?

I have done that already:
> AndreyMlashkin force-pushed the AndreyMlashkin:add_qt5_x11extras_module branch from 0b4c9e8 to 35aa5ac 3 days ago

but I can do that again

**Comment by jgsogo** [comment]
Hi! We are running some maintenance job in the CI right now. Most of the PRs have been triggered again so we have a very long queue of jobs waiting to build. This will take some time (and/or maybe it will fail just because of CI dying with so many jobs in the queue). 

I keep having a look at it all, and I will retrigger it manually if needed.

Thanks!

**Review by prince-chrismc — APPROVED** [review]

**Review by jgsogo — APPROVED** [review]

**Review by ericLemanissier — APPROVED** [review]

**Pull request merged**